atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/SweepResults.java

```diff
@@ -21,6 +21,7 @@
 
 import com.palantir.atlasdb.encoding.PtBytes;
 
+// Note that this class has an Auxiliary field, and therefore serialization might not work as expected
```

Comment:
I think it's fine. We don't specify the JSON serialize/deserialize annotations - and I did a quick test, and if we did the `timeSweepStarted` is still serialized.

Nice! I would have guessed such a feature must exist somewhere, but have never used it

Reply:
Oh cool. @gmaretic and I weren't sure if excluding the field from `toString()` would mess with `JsonSerialize` - thanks for doing the legwork here. I'll remove the comment, as it isn't needed.